NEWS

```diff
@@ -3,6 +3,22 @@
 History
 -------
 
+1.8.0 (YYYY-MM-DD)
+~~~~~~~~~~~~~~~~~~
+
+* Add support for JMESPath: you can now create a selector for a JSON document
+  and call ``Selector.jmespath()``. See :ref:`the documentation
+  <topics-selectors>` for more information and examples.
+* Memory usage improvements
```

Comment:
This might be misleading, because AFAIK memory usage of parsel should be exactly the same as in previous versions. 

What happened in https://github.com/scrapy/parsel/pull/269/ is that now it's possible to pass bytes + encoding to Selector, avoiding a need to create a unicode copy of the data, so there is an additional option for the parsel users to optimize the memory usage.